prometheus-exporters/vrops-exporter/alerts/vccluster.alerts

```diff
@@ -0,0 +1,34 @@
+groups:
+- name: vccluster.alerts
+  rules:
+  - alert: vCenterDRS_DisabledWithRegisteredVMs
```

Comment:
Name should start UpperCaseAndEverthingInThereLikeShouldBeWrittenLikeThis.
Have a look at our other prometheus alerts.